docs/Team/Database_Design_V2.md

```
@@ -0,0 +1,78 @@
+|-----|----------|-----------|-----------|----------|------|-------------------|------------------|------------------|
+
+*Composite index:* `(rider_id, vendor_id)`
+
+### pricing
+| id  | min_weight_gm | max_weight_gm | min_size_sqm | max_size_sqm | price    | created_at       | updated_at       |
+|-----|---------------|---------------|--------------|--------------|----------|------------------|------------------|
+
+*Ensure non-overlapping ranges for `min_weight_gm` and `max_weight_gm`.*
+
+
+### productcategory
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)